doc/doxygen/headers/boundary.h

```diff
@@ -79,64 +79,53 @@
- * describe the geometry of the triangulation.
- *
- * The behavior of the Triangulation class w.r.t. geometry
- * descriptions is the following: Triangulation::set_boundary() and
- * Triangulation::set_manifold() do the exact same thing: they attach
- * a manifold descriptor to the specified id. The first function
- * expects a Boundary descriptor (which is a specialization of a
- * Manifold description) and is provided mainly for backward
- * compatibility, while the second class expects a Manifold
- * descriptor. Notice that the Triangulation class only uses the
- * Manifold interface, and you could describe both the interior and
- * the boundary of the domain using the same object. The additional
- * information contained in the Boundary interface is related to the
- * computation of the exact normals. 
- *
- * Whenever a new vertex is needed in an object, the Triangulation
- * queries the manifold_id of the object which needs refinement. If
- * the manifold_id is set to numbers::invalid_manifold_id, then the
- * Triangulation queries the boundary_id (if the face is on the
- * boundary) or the material_id (if the Triangulation is of
- * codimension one and the object is a cell). If the previous queries
- * resulted in a number different from numbers::invalid_manifold_id,
- * then the Triangulation looks whether a previous call to
- * Triangulation::set_manifold() (or set_boundary()) was performed
- * with the given id, and if yes, it uses the stored object to obtain
- * new vertices, otherwise it uses a FlatManifold or StraightBoundary
- * object.
- *
- * @note This behavior is backward compatible to that of deal.II versions
- * prior to 8.2. If one ignores the manifold_id of an object (i.e., if it has
- * never been set), by default it is and remains set to
- * numbers::invalid_manifold_id. In that case, the first query above will
- * trigger a query to the old style boundary_id. This behavior will be
- * maintained for a while, but might eventually be changed. The suggested
- * strategy is to use manifold_ids to describe the geometry, and boundary_ids
- * to describe boundary conditions.
- * 
+ * Before version 8.2, the library allowed only boundary faces to follow a
+ * curved geometric description. Since version 8.2 this has been introduced also
+ * for interior faces and cells, and the boundary_id has been separated from the
+ * manifold_id. The former is used to identify the type of boundary conditions
+ * to apply, while the latter is used to identify the geometry and describe how
+ * new vertices should be created upon refinement of the mesh, or where high
+ * order Mapping objects should place their support points on the exact
+ * geometry.
+ *
+ * Since version 9.0 of the library, the boundary_id associated to the boundary
+ * faces is ignored by Manifold objects, and Manifold descriptors can only be
+ * attached to manifold ids.
+ *
+ * The behavior of the Triangulation class w.r.t. geometry descriptions is the
```

Comment:
w.r.t. -> with respect to

(I'm not sure how many of our non-proficient/non-native English users would understand the abbreviation)